corehq/apps/reports/models.py

```diff
@@ -80,6 +81,17 @@ class HQUserType(object):
     included_defaults = (True, True, True, True, False)
 
     @classmethod
+    def user_type_mapping(cls):
```

Comment:
Unfortunately no, it looks like we just save user_type for forms, it's in the [mapping file](https://github.com/dimagi/commcare-hq/blob/master/corehq/pillows/mappings/xform_mapping.py#L19-L19)